### PR TITLE
Add tutorial for creating a helper

### PR DESCRIPTION
**Pull request by StephenGregory**
What
---
Add a tutorial for creating your own helper, for those that want to learn how to use this repository to create their own.

### CODE REVIEW

File: docs/tutorial-pokemon.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+      tell Tasker this. More on this later.
+    */
+    return Tasker.leaveJavaScriptlet();
+}
+extractPokemonInformation(pokemonDetails, useShinyImage);
+```
+
+## 3. Test your script and iterate until it is ready
+
+If you did Step 1 and created a sandbox, you can now run `node sandbox/sandbox.tasker.pokemonInfoHelper.js`.
```
Comment (MitMaro) [inline docs/tutorial-pokemon.md:141]: Link to Step 1?

**Review by MitMaro — APPROVED** [review]

File: docs/tutorial-pokemon.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+    ![](assets/tutorial-pokemon/6.png)
+
+8. Your task should is now complete
+
+    ![](assets/tutorial-pokemon/7.png)
+
+9. Run your task:
+
+  ![](assets/tutorial-pokemon/8.png)
```
Comment (whymarrh) [inline docs/tutorial-pokemon.md:198]: The formatting of this image is a bit off, it isn't part of the bullet:

```suggestion
    ![](assets/tutorial-pokemon/8.png)
```

File: docs/tutorial-pokemon.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+function getTitle(pokemonDetails) {
+    const capitalizedName = pokemonDetails.name.charAt(0).toUpperCase() + pokemonDetails.name.slice(1);
+    return `#${pokemonDetails.id}: ${capitalizedName}`;
+}
+
+function getTypesLabel(types) {
+    if (!types) {
+        return '';
+    }
+    types.map(typeInfo => typeInfo.type.name).join(', ');
```
Comment (whymarrh) [inline docs/tutorial-pokemon.md:109]: Should this be `return`ed?

Comment (StephenGregory) [reply docs/tutorial-pokemon.md:109]: It should.

**Pull request merged**